service/helper.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 const (
-	transferGasLimit = uint64(21000) //nolint:gomnd
-	genesisTimestamp = 946713601000  // min allowable timestamp
+	nativeTransferGasLimit = uint64(21000) //nolint:gomnd
+	erc20TransferGasLimit  = uint64(100000)
```

Comment:
@patrick-ogrady , do you think this is high enough?

Reply:
That is too low